src/sentry/eventstream/kafka.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import absolute_import
```

Comment:
This is pretty much yanked directly from https://github.com/getsentry/getsentry/blob/master/getsentry/processing.py with a few modifications to be a service.

Reply:
👍